### ISSUE TITLE
Can I change Camera rotation?

### ISSUE THREAD

**Issue by sjg02122** [mmmmaomao/Carla0.9.10-kitti-data-export#5]
I just try Camera rotation 

        transform_sensor = carla.Transform(carla.Location(x=0, y=0, z=CAMERA_HEIGHT_POS))

to 

        transform_sensor = carla.Transform(carla.Location(x=0, y=0, z=CAMERA_HEIGHT_POS),carla.Rotation(10,0,0))

and I cann't get right 3d bounding boxes.

Do you have a idea about this?

**Comment by mmmmaomao** [comment]
Yes, you can do that, and it won't go wrong.May be you should check the bound_box file.
&nbsp;




------------------




王杰

北京邮电大学/本科生/现代邮政学院



13121815757

北京




&nbsp;
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: "Jinwoo Bae"; 
Date: 2021年4月25日(星期天) 下午2:41
To: "mmmmaomao/Carla0.9.10-kitti-data-export"; 
Cc: "Subscribed"; 
Subject: [mmmmaomao/Carla0.9.10-kitti-data-export] Can I change Camera rotation? (#5)

&nbsp;


 
I just try Camera rotation
     transform_sensor = carla.Transform(carla.Location(x=0, y=0, z=CAMERA_HEIGHT_POS))  
to
     transform_sensor = carla.Transform(carla.Location(x=0, y=0, z=CAMERA_HEIGHT_POS),carla.Rotation(10,0,0))  
and I cann't get right 3d bounding boxes.
 
Do you have a idea about this?
